### ISSUE TITLE
Number of Blocked On Assistant Icon 

### ISSUE THREAD

**Issue by bkrucarci** [AdguardTeam/AdguardAssistant#366]
### Feature Request
**OS:** Windows Latest
**Browser :** Opera Lastest
**AdGuard for Windows :** Latest

**I just love to see how many blocked on the website. The bare AdGuard extensions have this feature. I want it on AdGuard Assistant.**

![Screenshot_3](https://user-images.githubusercontent.com/15933805/145732009-709999af-0a07-45ab-bcc8-b44e2d27f8c3.png)

**Comment by zzebrum** [comment]
@bkrucarci thank you for the feature request! 
You've opened in the wrong repo and probably that's why you missed the fact that it's a duplicate of [#35](https://github.com/AdguardTeam/BrowserAssistant/issues/35).
